_pages/en_US/bannerbomb3.txt

```diff
@@ -22,43 +22,41 @@ If you have a Taiwanese console (indicated by a T after the system version), fol
 ### What You Need
 
 * Your `movable.sed` file from completing [Seedminer](seedminer)
+* The latest release of [BannerBomb3](https://github.com/zoogie/Bannerbomb3/releases/latest) (the Release `.zip` file)
 
 #### Section I - Prep Work
 
 In this section, you will copy the files needed to trigger the BannerBomb3 exploit onto your device's SD card.
 
-1. Open [BannerBomb3 Tool](https://3ds.nhnarwhal.com/3dstools/bannerbomb3.php) on your computer
-1. Upload your movable.sed using the "Choose File" option
-1. Click "Build and Download"
+1. Open [Bannerbomb3 Injector website](http://3dstools.nhnarwhal.com/#/bb3gen) on your computer
+1. Upload your movable.sed using the “Choose File” option
+1. Click “Build and Download”
+  + This will download an exploit DSiWare called F00D43D5.bin and other exploit data files inside of a zip archive (`unSAFE_MODE.zip`)
+1. Insert your SD card into your computer
+1. Navigate to `Nintendo 3DS` -> `<ID0>` -> `<ID1>` -> `Nintendo DSiWare` on your SD card
+  + `<ID0>` is the 32-letter folder name that you copied in [Seedminer](seedminer)
+  + `<ID1>` is a 32-letter folder inside of the `<ID0>`
   + This will download an exploit DSiWare called `F00D43D5.bin` inside of a zip file (`BannerBomb3.zip`)
 1. If your console is powered on, power off your console
 1. Insert your SD card into your computer
 1. Navigate to `Nintendo 3DS` -> `<ID0>` -> `<ID1>` on your SD card
   + `<ID0>` is the 32-letter folder name that you copied in [Seedminer](seedminer)
   + `<ID1>` is a 32-letter folder inside of the `<ID0>`
-  ![]({{ "/images/screenshots/bb3/dsiware-location-1.png" | absolute_url }}){: .notice--info}
 1. Create a folder named `Nintendo DSiWare` inside of the `<ID1>`
   + If you already had the folder *and* there are any existing DSiWare backup files (`<8-character-id>.bin`) inside, copy them to your PC and remove them from your SD card
-1. Copy the `F00D43D5.bin` file from `BannerBomb3.zip` to the `Nintendo DSiWare` folder
+1. Copy the `F00D43D5.bin` file from the `DSIWARE_EXPLOIT.zip` to the `Nintendo DSiWare` folder
+1. Copy `bb3.bin` from the `DSIWARE_EXPLOIT.zip` to the root of your SD card
+1. Reinsert the SD card into your device
 
-![]({{ "/images/screenshots/bb3/dsiware-location-2.png" | absolute_url }})
-{: .notice--info} 
 
-#### Section II - BannerBomb3
+#### Section II - Button Check
 
-In this section, you will trigger the BannerBomb3 exploit using the DSiWare Management menu and copy the resulting file dump to your computer so that you can use it on the next page.
```

Comment:
explainer: "In this section, we will check whether or not Safe Mode works as this will determine what exploit to use" or something along those lines

Reply:
Resolved.